README.md

```diff
@@ -1,10 +1,47 @@
-# Website
+# Project Layout
+
+### The project is broken down into 4 sections
+- Concepts - General Uniswap information or concepts useful for using Uniswap products, such as *Liquidity* and *Fees*
+- Contracts - The Uniswap smart contracts such as *Permit2*, or groupings such as the V3 Contracts
+- SDKs - The Uniswap SDKs such the *v3-sdk*
+- APIs - The Uniswap APIs such the *Subgraph API*
+
+### Each section contains items of that category
+For each item there should be:
+- Overview
+- Guides
+- Technical Reference
```

Comment:
These headers can be sentences instead of headers - it feels a bit odd to have long section names like this.